internal/env/env.go

```diff
@@ -0,0 +1,52 @@
+package env
+
+import (
+	"database/sql"
+	_ "github.com/lib/pq"
+	"github.com/vetal4ik10/lets-go-chat/configs"
+	"github.com/vetal4ik10/lets-go-chat/db/postgres"
+	"github.com/vetal4ik10/lets-go-chat/internal/models"
+	"log"
+)
+
+type Env struct {
+	db       *sql.DB
+	userRepo models.UserRepo
+}
+
+var env Env
+
+func init() {
```

Comment:
init not the best option for constructor. Every time when you import this function init is starting even if you don't need start  init function again